docs/examples/FormHorizontal.js

```diff
@@ -0,0 +1,37 @@
+const formInstance = (
+  <Form horizontal>
+    <FormGroup controlId="formHorizontalEmail">
+      <Col componentClass={ControlLabel} xs={2}>
+        Email
+      </Col>
+      <Col xs={10}>
+        <FormControl type="email" placeholder="Email" />
+      </Col>
+    </FormGroup>
+
+    <FormGroup controlId="formHorizontalPassword">
+      <Col componentClass={ControlLabel} xs={2}>
```

Comment:
this does kinda suck :/ I wish we had gone with an api for Col more like Collapse, where you had to provide a single child, that it cloned with the right classes...that's a whole other can of worms tho.

Perhaps it makes sense allow specifying Col props on the control-label since its such a common use case? I am not sure that isn't jsut confusing tho.